Updating Sysdig Classifier, Incident fields and Layout

S3B4SZ17 opened the pull request:
<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues


## Description
Updating the Sysdig default Classifier, Incident fields and Layout
Updated the overall docs and release notes to reflect current state and changes

## Must have
- [ ] Tests
- [X] Documentation

Benimanela reviewed (changes requested):
Hi @S3B4SZ17, Thank you for the work on this contribution. The implementation looks good overall.

### General
- Please run `demisto-sdk format` on all files to ensure consistency with XSOAR standards.

### Incident Fields
- Avoid changing the type of existing incident fields. This introduces breaking changes and may cause issues for users.
- Use common fields where possible instead of creating new ones. For example:
  - Use the common `Event ID` field instead of creating a new `Sysdig Event ID`.
  - Use the common `Category` field instead of `Sysdig Category`.

Let me know once you've addressed the feedback so I can review again!

S3B4SZ17 commented:
> @S3B4SZ17 i'll close the pr for lack of update, once you have some update feel free to reopen

Sorry, I have more time now and am addressing the above points to reopen the PR
